.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,33 @@
-For plugin updates, please indicate which repos this should be built into:
+If your package needs to be released within one or more release streams, and/or distributions, please open PRs to each of those branches respectively. The easiest way to do this is to make the initial commit for the mainline branch (e.g. rpm/develop or deb/develop) and then cherry pick the commit onto each subsequent branch.
 
-* [x] Nightly
-* [ ] 1.21
-* [ ] 1.20
+Supported Versions:
 
-See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.
+ * Nightly
+ * 1.21
+ * 1.20
+
+RPM Example:
+
+    git checkout -b rpm/develop-foreman-tasks-1.0.1 rpm/develop
+
+    # Make changes to update package
+
+    git commit -a -m 'Release foreman-tasks-1.0.1'
+    COMMIT=`git rev-parse HEAD`
+
+    git checkout -b rpm/1.20-foreman-tasks-1.0.1 rpm/1.20
+    git cherry-pick -x $COMMIT
```

Comment:
Would it be simpler to use `rpm/develop`?

Reply:
I don't understand the question sorry.

Reply:
Technically it could be `git cherry-pick -x rpm/develop-foreman-tasks-1.0.1` in this example. Not sure if that's more or less confusing so I'm fine either way.